src/__tests__/__snapshots__/test-utils-selectors.test.tsx.snap

```diff
@@ -430,8 +427,17 @@ Object {
     "awsui_constraint_1wzqe",
     "awsui_custom-control_1wzqe",
     "awsui_custom-filter-actions_1wzqe",
+    "awsui_dismiss-button_19bso",
     "awsui_remove-all_1wzqe",
+    "awsui_root_19bso",
     "awsui_root_1wzqe",
+    "awsui_select_19bso",
+    "awsui_token-content_19bso",
+    "awsui_token-editor-cancel_1wzqe",
+    "awsui_token-editor-field-operator_1wzqe",
+    "awsui_token-editor-field-property_1wzqe",
+    "awsui_token-editor-field-value_1wzqe",
+    "awsui_token-editor-submit_1wzqe",
```

Comment:
If you are moving so many class names around, how about introducing `property-filter/test-classes/styles.scss` file to settle them in a final location?

Reply:
These are new classnames mostly but I like the idea. I will add the test classes to the touched parts of the component